KatharaFramework/Kathara
# Aur

**- What I did**

I extended your packaging system for the aur. I wasn't able to make the compilation with `ccache` work. In the future it might be better to make the PKGBUILD download the compiled python script directly. For now, this is not a finished work.

Feel free to reject this pull request, then work briefly with me to resolve these issues.

**- How I did it**

I extended the `scripts/Linux-Pkg/Makefile`  to include the `aur` directive, so as long as the `.ssh/config` is configured correctly to push into the AUR it will work. Then I created `scripts/Linux-Pkg/kathara-aur`, which contains the specific files needed to install the package from source, including another Makefile.

**- How to verify it**

Verify the push function by cloning the repository, then

```bash
cd Kathara/scripts/Linkux-Pkg
make aur
```

Simulate an installation from the AUR by getting:

`Kathara/scripts/Linkux-Pkg/kathara-aur/*` and `Kathara/scripts/Linkux-Pkg/pacman/kathara.install`

Then

```bash
$ ls 
PKGBUILD kathara.install
$ makepkg -si
```

**- Description for the changelog**

* New workflow to push to the AUR
